test/acceptance/features/http.feature

```diff
@@ -43,3 +43,31 @@ Feature: HTTP client
           | json.inactive         | [FALSE]                   |
           | json.empty            | [EMPTY]                   |
           | json.null             | [NULL]                    |
+  
+  Scenario: Send a POST request defined by a json string
+    Given the HTTP endpoint "[CONF:url]/anything"
+      And the HTTP path "/test3"
+      And the HTTP request headers
+          | Authorization | Bearer access-token |
+      And the HTTP request body based in the JSON
+      """
+      {
+        "list": [
+          { "attribute": "attribute0", "value": "value0"},
+          { "attribute": "attribute1", "value": "value1"},
+          { "attribute": "attribute2", "value": "value2"}
+        ]
+      }
+      """
+     When I send a HTTP "POST" request
+     Then the HTTP status code must be "200"
+      And the HTTP response body must comply with the JSON schema "test-schema"
+      And I store the field value of the json HTTP response "json.list.2.attribute" in the context storage key "key"
+      And I store the field value of the json HTTP response "json.list.2.value" in the context storage key "value"
+      And the HTTP response body must have the JSON properties
+          | json.list.0.attribute | attribute0   |
+          | json.list.0.value     | value0       |
+          | json.list.1.attribute | attribute1   |
+          | json.list.1.value     | value1       |
+          | json.list.2.attribute | [CTXT:key]   |
+          | json.list.2.value     | [CTXT:value] |
```

Comment:
I think it's a bit weird to test it this way. I propose to:

1. Validate the reponse body normally:
`      And the HTTP response body must have the JSON properties
          | json.list.0.attribute | attribute0   |
          | json.list.0.value     | value0       |
          | json.list.1.attribute | attribute1   |
          | json.list.1.value     | value1       |
          | json.list.2.attribute | attribute2   |
          | json.list.2.value     | value2 |`
2. Then I store in context some keys, as in your example.
3. Send another HTTP request with this values from context.
4. Then  check again the response.

Reply:
OK.  Created a long scenario but agree that its more clear